pkg/exec/exec.go

```diff
@@ -37,15 +37,22 @@ var (
 )
 
 func RunCommandInOperatorPod(ctx context.Context, clientsets *k8sutil.Clientsets, cmd string, args []string, operatorNamespace, clusterNamespace string, exitOnError bool) string {
+	var pod v1.Pod
+	var err error
 
-	pod, err := k8sutil.WaitForPodToRun(ctx, clientsets.Kube, operatorNamespace, "app=rook-ceph-operator")
+	pod, err = k8sutil.WaitForPodToRun(ctx, clientsets.Kube, operatorNamespace, "app=rook-ceph-operator")
 	if err != nil {
 		logging.Fatal(fmt.Errorf("failed to wait for operator pod to run: %v", err))
 	}
 
 	var stdout, stderr bytes.Buffer
 
-	err = execCmdInPod(ctx, clientsets, cmd, pod.Name, "rook-ceph-operator", pod.Namespace, clusterNamespace, args, &stdout, &stderr)
+	if exitOnError {
```

Comment:
This condition seems backwards if I'm reading it correctly
```suggestion
	if !exitOnError {
```

Reply:
Or maybe i'm just confused by what it means that the method is named "without return", and then line 52 returns.